lib/grpc/message.ex

```diff
@@ -47,7 +48,13 @@ defmodule GRPC.Message do
       {:error, "Encoded message is too large (#{length} bytes)"}
     else
       result = <<compress_flag, length::size(4)-unit(8), message::binary>>
-      {:ok, result, length + 5}
+
+      data =
+        if function_exported?(codec, :pack_for_channel, 1),
```

Comment:
please update with master because this function has changed. We also need to ensure that the coded can handle the new possible iodata format instead of just binaries in pack_for_channel

Reply:
let me know if I should do this instead :)

Reply:
Updated to match master :). So `pack_for_channel` should accept `iodata` and return a `iolist` instead of a binary?
Or do you think the function should maintain the same shape as the input type? i.e `iolist -> iolist | binary -> binary`. Sort of:

```elixir
def pack_for_channel(data) when is_list(data) do
  data
  |> IO.iodata_to_binary()
  |> Base.encode64()
  |> List.wrap()
end
```